samples/csharp/common/AutoML/ConsoleHelper.cs

```diff
@@ -55,6 +55,15 @@ public static void PrintMulticlassClassificationMetrics(string name, MulticlassC
             Console.WriteLine($"************************************************************");
         }
 
+        public static void PrintRankingMetrics(string name, RankingMetrics metrics)
+        {
+            Console.WriteLine($"************************************************************");
+            Console.WriteLine($"*    Metrics for {name} ranking model   ");
+            Console.WriteLine($"*-----------------------------------------------------------");
+            Console.WriteLine($"    Discounted Cumulative Gains = {metrics.DiscountedCumulativeGains.Average():0.####}");
+            Console.WriteLine($"    Normalized Discounted Cumulative Gains = {metrics.NormalizedDiscountedCumulativeGains.Average():0.####}, a value between 0 and 1, the closer to 1, the better");
```

Comment:
There's a few places still using the `.Average()`, which should print one of the values of the array. Another choice is to print all.

```suggestion
            Console.WriteLine($"    Normalized Discounted Cumulative Gain (NDCG@10) = {metrics?.NormalizedDiscountedCumulativeGains[9] ?? double.NaN:0.####}, a value between 0 and 1, the closer to 1, the better");
```